pkg/rpc/server/inject.go

```diff
@@ -0,0 +1,24 @@
+// +build wireinject
+
+package server
+
+import (
+	library2 "github.com/aquasecurity/trivy/pkg/rpc/server/library"
```

Comment:
why do we need aliases here? the actual name should work fine right?

Reply:
You're right. IDE added the alias automatically. I fixed it.
https://github.com/aquasecurity/trivy/pull/307/commits/99d6f5ad15cfc90143f7ba8edd43301ecf247fd4